web/admin/src/store/coordinator.ts

```diff
@@ -290,5 +292,67 @@ export const useCoordinatorStore = defineStore('Coordinator', {
       }
       this.fetchCoordinators()
     },
+
+    /**
+     * コーディーネータに生産者を紐づける非同期関数
+     * @param id 生産者を紐づけるコーディネータのID
+     * @param payload コーディネーターに紐づく生産者
+     * @returns
+     */
+    async relateProducers(
+      id: string,
+      payload: RelateProducersRequest
+    ): Promise<void> {
+      try {
+        const authStore = useAuthStore()
+        const accessToken = authStore.accessToken
+        if (!accessToken) {
+          return Promise.reject(new Error('認証エラー'))
+        }
+        const factory = new ApiClientFactory()
+        const contactsApiClient = factory.create(CoordinatorApi, accessToken)
+        await contactsApiClient.v1RelateProducers(id, payload)
+        const commonStore = useCommonStore()
+        commonStore.addSnackbar({
+          message: 'コーディネーターと生産者の紐付けが完了しました',
+          color: 'info',
+        })
+      } catch (error) {
+        if (axios.isAxiosError(error)) {
```

Comment:
共通のエラーハンドラー使ってもらっても良さそう👀

```suggestion
        return this.errorHandler(error)
```

Reply:
これってこんな機能あったんだね、笑
共通のエラーハンドラーあったのか、ちょっと描き直しとく、
あざます！